### PR TITLE
Enables y-scroll on textarea resize - Fixes issue 132

### PR DESCRIPTION
**Pull request by share-with-me**
This PR fixes issue #132 . 
As discussed on the issue thread, currently, on resizing the `originalText` div using the resize drag handler, if the height of the div decreases as compared to the text contents, `y-scroll` does not appear making it difficult for the user to view all the contents. This PR fixes it.

### CODE REVIEW

**Comment by sushain97** [comment]
For some reason, Travis isn't importing things here. You can always view them at https://travis-ci.org/goavki/apertium-html-tools/pull_requests though.

**Comment by share-with-me** [comment]
@sushain97 , The Travis build succeeds now :)

**Comment by share-with-me** [comment]
Reviews on this @sushain97 ?!

File: assets/js/util.js [hunk 1]
```diff
@@ -162,6 +164,12 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
+        if($('#originalText').height() < originalTextScrollHeight) {
+            $('#originalText').css('overflow-y', 'scroll');
+        }
+    });
+
```
Comment (sushain97) [inline assets/js/util.js]: extra line?

File: assets/js/util.js [hunk 2]
```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
+        if($('#originalText').height() < originalTextScrollHeight) {
+            $('#originalText').css('overflow-y', 'scroll');
```
Comment (sushain97) [inline assets/js/util.js]: Why can't this situation be solved with simply a `overflow: y-scroll`? I'm guessing you've tried it.

File: assets/js/util.js [hunk 3]
```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
```
Comment (sushain97) [inline assets/js/util.js]: Why the `mouseup` event? Is there not a resize event?

File: assets/js/util.js [hunk 4]
```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
...
+        if($('#originalText').height() < originalTextScrollHeight) {
```
Comment (sushain97) [inline assets/js/util.js]: Is it possible to check something like `:overflow`?

File: assets/js/util.js (hunk 2 above)
Comment (share-with-me) [inline assets/js/util.js]: > Why can't this situation be solved with simply a `overflow: y-scroll`?

Um, I had tried that! As soon as `synchronizeTextareaHeights` method is called, the `CSS` for this div changes. And thus, I started with this approach of checking if the `textarea` height falls in the range for scrolling!

File: assets/js/util.js (hunk 3 above)
Comment (share-with-me) [inline assets/js/util.js]: > Is there not a resize event?

This [stackoverflow link](https://stackoverflow.com/questions/5570390/resize-event-for-textarea) discusses about a `resize` event that we will need to code up to detect a change in `textarea` dimensions. This too, uses `mouseup` event only! Thus, I chose to go with the `mouseup` approach!

File: assets/js/util.js (hunk 4 above)
Comment (share-with-me) [inline assets/js/util.js]: Um, `:overflow` to check if the element is having the `overflow`?

**Comment by share-with-me** [comment]
Damn! Did a git pull and rebase and it gave me this!

**Comment by sushain97** [comment]
You shouldn't be doing a pull and rebase. You should be doing `git pull --rebase`.

**Comment by share-with-me** [comment]
Ohh, I will try it now!

**Comment by share-with-me** [comment]
I have the solution ready for issues #132 and #151  in another branch. Will be easier for me to create a new PR for issue 151 which will handle both :P. Can I go ahead with that? :P

**Comment by sushain97** [comment]
> Will be easier for me to create a new PR for issue 151 which will handle both :P. Can I go ahead with that? :P

Sure.

**Comment by share-with-me** [comment]
Thanks :D .
Closing this and continuing it in #152

**Pull request closed without merging**